PWGCF/FemtoDream/FemtoDreamTrackSelection.h

```diff
@@ -344,63 +344,18 @@ void FemtoDreamTrackSelection::init(HistogramRegistry* registry, const std::stri
 template <typename T>
 auto FemtoDreamTrackSelection::getNsigmaTPC(T const& track, o2::track::PID pid)
 {
-  switch (pid) {
-    case o2::track::PID::Electron:
-      return track.tpcNSigmaEl();
-      break;
-    case o2::track::PID::Muon:
-      return track.tpcNSigmaMu();
-      break;
-    case o2::track::PID::Pion:
-      return track.tpcNSigmaPi();
-      break;
-    case o2::track::PID::Kaon:
-      return track.tpcNSigmaKa();
-      break;
-    case o2::track::PID::Proton:
-      return track.tpcNSigmaPr();
-      break;
-    case o2::track::PID::Deuteron:
-      return track.tpcNSigmaDe();
-      break;
-    default:
-      return 999.f;
-      break;
-  }
+  return o2::aod::pidutils::tpcNSigma(pid, track);
```

Comment:
This is not fully equivalent as in this case a Nsigma is returned even for IDs greater than deuterons